job-templates/kubernetes_release_staging.json

```
@@ -9,8 +9,8 @@
       "orchestratorRelease": "1.17",
```

Should we update this while we are making updates?
Same with kubernetes_release_staging_serial.json?

---

Or leave it blank like in kubernetes_containerd_master_serial.json  and a few others?

---

```suggestion
      "orchestratorRelease": "",
```

Sure it get overridden in the prow jobs but that makes it less confusing

---

updated